backend/cms/apps.py

```diff
@@ -1,12 +1,17 @@
 """
 Django related class representing a config of an app
 """
+import sys
+from django.conf import settings
 from django.apps import AppConfig
 
-
 class CmsConfig(AppConfig):
     """
     Class inheriting the django AppConfig
     """
 
     name = 'cms'
+
+    if settings.SECRET_KEY == '-!v282$zj815_q@htaxcubylo)(l%a+k*-xi78hw*#s2@i86@_' and not settings.DEBUG:
+        print("Your are running the Integreat CMS in production mode. Change the SECRET_KEY in the settings.py!")
```

Comment:
Just a small thing, but here's a typo... Should be "you" instead of "your".
Apart from that, do we want to hard code the same string into the code twice (in `settings.py` and in `apps.py`) or should we move that to a constant?

Reply:
Done